HACKING.md

```diff
@@ -6,6 +6,21 @@ These instructions will cover both local developer workflows, as well as
 interacting with the production pipeline in CentOS CI. A section header
 may indicate that the section only applies to one of the two workflows.
 
+Another option if one simply wants to test a different COSA image or
+FCOS config or pipeline code (but otherwise maintain the manifest the
+same) is to create a developer pipeline alongside the production one. To
+do this, you *must* use the `devel-up` script. For example:
+
+```
+./devel-up \
+        --pipeline https://github.com/jlebon/fedora-coreos-pipeline \
+        --config https://github.com/jlebon/fedora-coreos-config@feature \
+        --cosa-img quay.io/jlebon/coreos-assembler:random-tag
+```
```

Comment:
`devel-up: error: one of the arguments --update --delete is required`

Reply:
Thanks, fixed!